decks/service-worker.mdx

```diff
@@ -0,0 +1,147 @@
+import { Head } from 'mdx-deck'
+
+export { default as theme } from '../theme'
+
+<Head>
+  <title>Service Worker</title>
+  <meta name="author" content="Tobias Linkohr" />
+</Head>
+
+# Service Worker
+
+by Tobias Linkohr  
+([github](https://github.com/Tobi-mmt))
+
+
+---
+
+# What is it?
+programmable network proxy
+
+---
+
+![](./images/service-worker/sw.jpeg)
```

Comment:
Could you change this to `<img src="https://.." />` please. This is needed for the build.